testsuite/features/support/xmlrpc_system.rb

```diff
@@ -95,4 +95,11 @@ def list_empty_system_profiles
   def obtain_reactivation_key(server)
     @connection.call('system.obtainReactivationKey', @sid, server)
   end
+
+  # Namespace system.search
+  # Provides methods to perform system search requests using the search server
+
```

Comment:
I think you can skip the additional blank line here according to the rest of the comments in this class.